code/modules/clothing/suits/armor.dm

```diff
@@ -193,5 +193,4 @@
 /obj/item/clothing/suit/armor/tdome/green
 	name = "thunderdome suit"
 	desc = "Pukish armor."	//classy.
-	icon_state = "tdgreen"
-	item_state = "tdgreen"
```

Comment:
Nothing changed, it was just that I accidentally deleted the line when I was finishing my commits and readded it.